### ISSUE TITLE
Accessibility issues in the demo of Drum Machine

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/freeCodeCamp#63033]
## Description

The demo of the Drum Machine lab has the following a11y issues:
- Poor color contrast between the pads and the container (`#444` against `#3b3b5f`)
- The pads are interactive so they should be `button`s rather than `div`s
- The power toggle should be an `<input type="checkbox">` rather than a `div`
- The volume slider's `input` should have an associated label

<details>
  <summary>Screenshot</summary>   
    
  <img width="894" height="686" alt="Image" src="https://github.com/user-attachments/assets/d96b6ca5-cc41-4657-8580-7a83363fcb2f" />

</details>

---

For the lab itself, there should be an additional test to ensure the pads are displayed as buttons. The power toggle and volume slider aren't part of the requirements, so no tests are needed.

## Affected page

https://www.freecodecamp.org/learn/full-stack-developer/lab-drum-machine/build-drum-machine

**Comment by Om7035** [comment]
can i work on this ??
